packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -0,0 +1,46 @@
+.grav-c-list-cards-basic {
+  $breakpoint-1: gravy-breakpoint(medium);
+  $breakpoint-2: gravy-breakpoint(large);
+  padding: 0;
+  list-style-type: none;
+
+  @media all and (min-width: $breakpoint-1) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  > li + li {
+    margin-top: $grav-sp-xl;
+  }
+
+
+  @media all and (min-width: $breakpoint-1) {
+    > {
+      // 2 col layout
+      li {
+        display: flex;
+        flex: 50%;
+        padding: 0 $grav-sp-l;
+      }
+
+      // Remove top margin from 2nd card
+      // since it appears at top of list now
+      li:nth-child(2) {
+        margin-top: 0;
+      }
+
+      // Callouts go full-width
+      .grav-c-list-cards-basic-callout {
```

Comment:
Hmm.. given that we use BEM-ish conventions, I'd say this callout class ought to be an element modifier

I'd argue the `<li>` is an _element_ of the `.grav-c-list-cards-basic` _block_, and this is a _modifier_ of that element.

So, even though it's a bit verbose, maybe `.grav-c-list-cards-basic--item__callout` is more correct.

Alternatively, I suppose you could argue the "callout" is an _element_ of the list (just one that is not always used) and there make the class `.grav-c-list-cards-basic--callout`.

I'll let you choose which you prefer.

is either an element (the `<li>`s it's applied to are after all _elements_ of the `.grav-c-list-cards` _block_) or a modifier (as in, it _modifies_